lib/rules/string-no-newline/README.md

```diff
@@ -70,3 +70,33 @@ a {
   font-family: "Times New Roman";
 }
 ```
+
+## Optional secondary options
+
+### `ignore: ["at-rules", "declarations"]`
+
+#### `"at-rules"`
+
+Ignore strings in at-rules.
```

Comment:
minor
pedantic

I don't have a proposition, but based on the example

```css
@import url('example
.com');
```

the current description could be interpreted differently.
i.e. I would expect _in_ in "[…] **in** at-rules." to refer to the scope of a block of a [block at-rule](https://www.w3.org/TR/css-syntax-3/#ref-for-block-at-rule).

I don't think renaming the option to be `statement-at-rules` is necessary though.

Reply:
You raise a good point. These options help users ignore overlaps in our rules that check at-rule _preludes_ and declaration _values_. Let's be accurate in naming so that the connection is clearer to users.

We've typically favoured accuracy in our other options, even if it's more verbose.

Let's rename the options to:

- `at-rule-preludes` - Ignore strings in at-rule preludes.
- `declaration-values` - Ignore strings in declaration values.